test/Microsoft.AspNetCore.Mvc.FunctionalTests/ContentNegotiationTest.cs

```diff
@@ -345,33 +345,31 @@ public ContentNegotiationTest(MvcTestFixture<BasicWebSite.Startup> fixture)
         }
 
         [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task ObjectResult_WithStringReturnType_DefaultToTextPlain(bool matchFormatterOnObjectType)
+        [InlineData(null)]
+        [InlineData("text/plain")]
+        [InlineData("text/plain; charset=utf-8")]
+        [InlineData("text/html, application/xhtml+xml, image/jxr, */*")] // typical browser accept header
+        public async Task ObjectResult_WithStringReturnType_DefaultToTextPlain(string acceptMediaType)
         {
             // Arrange
-            var targetUri = "http://localhost/FallbackOnTypeBasedMatch/ReturnString?matchFormatterOnObjectType=true" +
-                matchFormatterOnObjectType;
```

Comment:
> Did this test previously send ...?matchFormatterOnObjectType=truetrue and ...?matchFormatterOnObjectType=truefalse?

The action wasn't using this value. It appears an earlier clean up wasn't complete.